packages/core/src/fetch/__tests__/json.response.data.test.ts

```diff
@@ -52,7 +52,12 @@ describe('fetch/json.response.data', () => {
       params: { body: {} },
     });
 
-    expect(watcher.listeners.onDoneData).toBeCalledWith({ test: 'value' });
+    expect(watcher.listeners.onDoneData).toBeCalledWith({
```

Comment:
🤓 in my opinion, `{ result, meta: { headers } }` would be better to mirror the final structure in _Queries_